oblt-cli/cluster-credentials/action.yml

```diff
@@ -0,0 +1,41 @@
+name: 'oblt-cli/cluster-credentials'
+description: 'Run the oblt-cli wrapper to retrieve the credentials to connect to the given cluster'
+inputs:
+  cluster-name:
+    description: 'The cluster name'
+    required: false
+  cluster-info-file:
+    description: 'The cluster info file (absolute path)'
+    required: false
+  github-token:
+    description: 'The GitHub access token.'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - uses: elastic/oblt-actions/oblt-cli/cluster-name-validation@v1.2.0
```

Comment:
what's our plan for this? major only?

Reply:
I chose this because the `oblt-cli/cluster-name-validation` was released in v1.2.0

Reply:
My thought was that if we want reproducible workflows, we might need to pin this to the exact version.

Reply:
At some point, these versions might get aligned using Dependabot.
But using the exact version will help with preserving the same behaviour between versions.

Reply:
so lets say 1.10.0 is still using 1.5.0 in some internal composite action.

when dependabot upgrades the usage to 1.10.0 the next release (e.g 1.10.1) will have the pinned versions.

hence, if we go back from 1.10.1 to 1.10.0 we are sure it will behave the same way as before.